src/test/java/seedu/address/logic/listcommands/ListCompaniesCommandTest.java

```diff
@@ -31,10 +29,4 @@ public void setUp() {
     public void execute_listIsNotFiltered_showsSameList() {
         assertCommandSuccess(new ListCompaniesCommand(), model, ListCompaniesCommand.MESSAGE_SUCCESS, expectedModel);
     }
-
-    @Test
-    public void execute_listIsFiltered_showsEverything() {
-        showPersonAtIndex(model, INDEX_FIRST_PERSON_OR_COMPANY);
-        assertCommandSuccess(new ListCompaniesCommand(), model, ListCompaniesCommand.MESSAGE_SUCCESS, expectedModel);
-    }
```

Comment:
Sorry I know I said we could consider deleting this but I saw the same test in the ListPersonCommandTest so we should probably keep it? What do you think?

Reply:
Rongzhi deleted this test as it could not be passed, probably because Hathu recently changed some names. He promised he would add it back later. Let's merge this first.